documentation/general/dotnet-run-file.md

```diff
@@ -178,7 +178,9 @@ have the shared `.cs` files source-included via `<Compile Include="../Shared/**/
 
 ## Build outputs
 
-Build outputs are placed under a subdirectory whose name is hashed file path of the entry point
+If [artifacts output layout][artifacts-output] is enabled, build outputs of the file-based app are placed there
+(except caching markers which are placed in the global temp directory described next).
```

Comment:
I admit I'm not understanding the motivation of this change here. Do we expect users to then be looking at these outputs, or is this just to leverage the existing things that exclude artifact outputs?

Reply:
Will the artifacts at least be placed under a `runfile` subfolder or some such? Otherwise I worry a bit that the artifacts directory will be polluted.

In particular, if a file is not part of a loaded project (e.g. it was not loaded yet or it was deliberately unloaded), the IDE currently will just treat it as a file-based program in many cases, it will do a design time build on it in order to get basic set of references and so on. It could be confusing if the artifacts folder contains many such directories simply due to the way the user was browsing.